_projects/04-sip-process-reloaded.md

```diff
@@ -9,7 +9,7 @@ status: Active
 type: project
 active: true
 priority: 1
-category: process
+category: enhancement
```

Comment:
I’m not sure about this one, indeed the SIP _is_ a process, but I see it as a way to enhance the language, hence the category `enhancement`

Reply:
I agree that `enhancement` is better for SIP.